org.OpenT2T.Onboarding.LightifyHub/org.OpenT2T.Onboarding.LightifyHub.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE node PUBLIC "-//freedesktop//DTD D-BUS Object Introspection 1.0//EN" "http://standards.freedesktop.org/dbus/introspect-1.0.dtd"[]>
+<!--  This is the metadata for a category of things that are connected via the Wink Hub. -->
+<node>
+    <interface name="com.OpenT2T.Onboarding.LightifyHub">
+        <!-- Defines the metadata required to start onboarding.
+             These input parameters are unique to a particular device type that supports this onboarding type,
+             e.g. auth URLs, RF bands, regular expressions to find the device on a common bus, etc. -->
+        <method name="onboard">
+            <arg name="device_type_filter" type="s" direction="in" />
+        </method>
+        <!-- Defines the metadata returned by onboarding. This is converted to the props array on the device
+             object, which is passed in to translators that implement this onboarding type at runtime. -->
+        <signal name="success">
+            <arg name="security_token" type="s" direction="in" />
+            <arg name="device_id" type="s" direction="in" />
+            <arg name="message" type="s" direction="in" />
+        </signal>
```

Comment:
This is a general comment that applies to all onboarding types, and should not be fixed here until we resolve the overall design issue and update other components accordingly...

Signals should not be used for async return values or async errors that result from calling a method. Successful results should be declared as one or more out parameters on the method itself (which is implicitly async). Errors should be reported as exceptions (rejected promises) from the method.